addons/script-snap/addon.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "Snap scripts to grid",
+  "description": "Drag a script to automatically align its position to the workspace dots.",
```

Comment:
We don't ever use the word "workspace" user facing, so this should be "code area" or something else.

Reply:
Yeah, it's on https://scratchaddons.com/docs/reference/addon-manifest/